app/(auth)/verification/page.tsx

```diff
@@ -2,13 +2,14 @@
 
 import { useEffect, useState } from 'react';
 import { useRouter } from 'next/navigation';
-import supabase from '@/api/supabase/createClient';
 import {
   getTempEmail,
   resendVerificationEmail,
 } from '@/api/supabase/queries/auth';
 import Bud from '@/public/images/bud.svg';
 import EmailIcon from '@/public/images/email.svg';
+// import supabase from '@/api/supabase/createClient';
+import { useSession } from '@/utils/AuthProvider';
 import {
```

Comment:
replace `../../../styles/styles` to `@/styles/styles`

Reply:
fixed in latest commit!